packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -39,6 +39,10 @@ import { OfficeGraphInsightString, ViewType } from '../../graph/types';
 import { styles } from './mgt-file-list-css';
 import { strings } from './strings';
 import { MgtFile } from '../mgt-file/mgt-file';
+import { FluentDesignSystemProvider, FluentProgressRing } from '@fluentui/web-components';
+
+FluentDesignSystemProvider;
+FluentProgressRing;
```

Comment:
I noticed the fluent components don't get bundled in this way. But it looks like if instead of importing, we re-export the components, then they will get picked up and bundled.

```suggestion

export { FluentDesignSystemProvider, FluentProgressRing } from '@fluentui/web-components';
```